portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,12 +27,29 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final ImmutableList<String> immutableList = ImmutableList.of("Mango","Strawberry","Banana");
-  Random rand = new Random();
+  private ArrayList<String> commentsList = new ArrayList();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
```

Comment:
this should be application/json
https://stackoverflow.com/questions/477816/what-is-the-correct-json-content-type

Reply:
Noted, thank you